WPMedia/fusion-news-theme-blocks
# [Themes 1212] Added logical properties for numbered-list, card-list, top-table-list and simple-list blocks

## Description

Update padding, margin, and border properties. The following conversions must be done:

1. *-top and *-bottom need to be changed to *-block
2. *-left and *-right need to be changed to *-inline
3. height and width will need to be set to block-size and inline-size respectively

## Jira Ticket

- [THEMES-1212](https://arcpublishing.atlassian.net/browse/THEMES-1212)

## Acceptance Criteria

1. The above logical properties are applied and tested on the list blocks.
2.Storybook is updated with RTL.

## Test Steps

1. Checkout this branch `git checkout {branch name}`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/simple-list-block,@wpmedia/top-table-list-block,@wpmedia/numbered-list-block,@wpmedia/card-list-block`
3. Set "textDirection": "rtl" in the "siteProperties" object of the blocks.json
4. content source `_id: PKFVHJKBBVHZRIHJEMQGGREWIE`
5. Check the styles for RTL matching to Figma designs



## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1212]: https://arcpublishing.atlassian.net/browse/THEMES-1212?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
